src/Tools/Source/CompilerGeneratorTools/Source/CSharpSyntaxGenerator/CSharpSyntaxGenerator.csproj

```diff
@@ -8,10 +8,15 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>Roslyn.Compilers.CSharp.Internal.CSharpSyntaxGenerator</RootNamespace>
     <AutoGenerateBindingRedirects>True</AutoGenerateBindingRedirects>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Updated so i could use ranges.  If this is unpalatable it would be easy to continue using 2.1 and just manually writing slices myself.

Reply:
This is fine. Our repository allows for non-shipping binaries to target netcoreapp3.0

---
In reply to: [312250862](https://github.com/dotnet/roslyn/pull/37840#discussion_r312250862) [](ancestors = 312250862)

Reply:
Sweet.  Thanks @jaredpar !